File: DSA/ShatakshiMishra_590015008/Day8Question1.cpp
#include <iostream>
#include <string>
#include <algorithm>

using namespace std;

int main() {
    string input;
    
    getline(cin, input);

    int length = input.length();

    string upper = input;
    transform(upper.begin(), upper.end(), upper.begin(), ::toupper);

    string lower = input;
    transform(lower.begin(), lower.end(), lower.begin(), ::tolower);

    cout << "Length: " << length << ", ";
    cout << "Original: \"" << input << "\"" << ", ";
    cout << "Uppercase: \"" << upper << "\"" << ", ";
    cout << "Lowercase: \"" << lower << "\"" << endl;
}

File: DSA/akashyadav_590014126/Day_8_Question_1.py
text = input("Enter a string: ")#taking input by the user

length = len(text)#calculating the length

uppercase = text.upper()#converting to uppercase
lowercase = text.lower()#converting to lowercase

# Output
print(f"Length: {length}, Original: \"{text}\", Uppercase: \"{text}, Lowercase: \"{text}\"")